backend-service/Embedded/Relay.py

```diff
@@ -21,25 +21,25 @@ def __init__(self, id, input_state):
         #set GPIO pin as output pin
         GPIO.setup(self.pin_id, GPIO.OUT)
         #set the GPIO state to either high or low
-        GPIO.output(self.pin_id, GPIO.HIGH if self.state else GPIO.LOW)
+        GPIO.output(self.pin_id, GPIO.LOW if self.state else GPIO.HIGH)
```

Comment:
Because we are controlling the Relay, when it's HIGH that means we are 'closing' up the circuit used by our devices.